scripts/integration.sh

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env bash
```

Comment:
Also not sure about this one here. Currently we don't have any integration tests I'd say.

Reply:
removed